universal-login-relayer/lib/integration/ethereum/validators/EnoughTokenValidator.ts

```diff
@@ -10,24 +10,24 @@ const isContract = async (provider: providers.Provider, contractAddress: string)
   return isContractExist(bytecode);
 };
 
-export const hasEnoughToken = async (gasToken: string, walletContractAddress: string, gasLimit: utils.BigNumberish, provider: providers.Provider) => {
+export const hasEnoughToken = async (gasToken: string, walletContractAddress: string, gasLimit: utils.BigNumberish, gasPrice: utils.BigNumberish, provider: providers.Provider) => {
```

Comment:
Consider two additional refactorings:
- extract isContract to separate files
- move EnoughTokenValidator to commons